lib/racecar/config.rb

```diff
@@ -66,14 +66,53 @@ class Config < KingKonf::Config
     desc "The log level for the Racecar logs"
     string :log_level, default: "info"
 
+    desc "Protocol used to communicate with brokers"
+    symbol :security_protocol, allowed_values: %i{plaintext ssl sasl_plaintext sasl_ssl} + %w{plaintext ssl sasl_plaintext sasl_ssl}
```

Comment:
Why allow strings here?

Reply:
Just convenience, because I found [a piece of documentation](https://github.com/zendesk/racecar#producing-messages-1) describing the config value being either string or symbol. Since rdkafka does not care I thought about rather being consistent and allowing both.

Reply:
I think KingKonf should automatically cast the value to a Symbol in this case, though – is that not the case?

Reply:
Let me fix that in KingKonf.

Reply:
Will have a fix ready Monday.

Reply:
How does this look? https://github.com/dasch/king_konf/pull/14